Packs/CommonPlaybooks/Playbooks/playbook-Ticket_Management_-_Generic.yml

```diff
@@ -0,0 +1,703 @@
+                value:
+                  simple: There is no hostname in the alert
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 440,
+          "y": 760
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "16":
+    id: "16"
+    taskid: c8d353d1-1319-420f-8831-8d04cb40effd
+    type: regular
+    task:
+      id: c8d353d1-1319-420f-8831-8d04cb40effd
+      version: -1
+      name: Set key to open ticket
+      description: Set a value in context under the key you entered.
+      scriptName: Set
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "18"
+    scriptarguments:
+      key:
+        simple: Action
+      value:
+        simple: NewTicket
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 640,
+          "y": 40
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "17":
+    id: "17"
+    taskid: 2e759af3-962e-48ad-8493-c4052d84f866
+    type: regular
+    task:
+      id: 2e759af3-962e-48ad-8493-c4052d84f866
+      version: -1
+      name: Set key to add a comment
+      description: Set a value in context under the key you entered.
+      scriptName: Set
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "18"
+    scriptarguments:
+      key:
+        simple: Action
+      value:
+        simple: AddComment
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 210,
+          "y": 40
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "18":
+    id: "18"
+    taskid: 0be81a91-6f8f-488c-8cdc-9df4684fe290
+    type: title
+    task:
+      id: 0be81a91-6f8f-488c-8cdc-9df4684fe290
+      version: -1
+      name: Untitled Task
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "2"
+      - "3"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 430,
+          "y": 210
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "19":
+    id: "19"
+    taskid: 043ae1ef-de8c-490b-8526-cb5fdd8f521b
+    type: condition
+    task:
+      id: 043ae1ef-de8c-490b-8526-cb5fdd8f521b
+      version: -1
+      name: Open a new ticket or add a comment
```

Comment:
Maybe change to "Open a new ticket or comment on existing"

Reply:
Done